app-emulation/wine-proton/wine-proton-7.0.4.ebuild

```diff
@@ -195,6 +206,10 @@ src_configure() {
 		$(use_with vkd3d)
 		$(use_with xcomposite)
 		$(use_with xinerama)
+
+		# tests are non-trivial to run, can hang easily, don't play well with
+		# sandbox, and several need real opengl/vulkan or network access
+		--disable-tests
```

Comment:
should we also have RESTRICT=test?

Reply:
Could but fwiw there's no tests to run right now so restricting does nothing, it'd just mean moving/duplicating the explanation